src/include/rados/rados_types.h

```diff
@@ -8,14 +8,17 @@
  * One item from list_watchers
  */
 struct obj_watch_t {
+  /// Address of the Watcher
```

Comment:
just want to note that `//` one-line comment was introduced by C99, and is also supported by GCC and clang extension, but if a user is using `-std=c89 -pedantic` for building his/her librados client, it won't compile with this change. by default both GCC and clang use C11 if `-std` is not specified.

Reply:
thanks for checking! in that case, we don't need to worry about C89 users.